requirements.dev.txt

```diff
@@ -38,3 +38,4 @@ pyright==1.1.351
 python-json-logger==2.0.7  # unpinned transitive dependency of jupytext breaking on later versions
 console-ctrl==0.1.0
 synchronicity[compile]==0.10.3
+googleapis-common-protos~=1.72.0
```

Comment:
Looks like these protos were build with 4.25 which is newer than the minimal version we test on?

Reply:
may just need to conditionally skip some tests?